lxd/db/images.go

```diff
@@ -7,12 +7,11 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/pkg/errors"
-
 	"github.com/lxc/lxd/lxd/db/query"
 	"github.com/lxc/lxd/lxd/instance/instancetype"
 	"github.com/lxc/lxd/shared/api"
 	"github.com/lxc/lxd/shared/osarch"
+	"github.com/pkg/errors"
```

Comment:
This change is un-needed and incorrect, the current order is the expected one for LXD Go files.